### DISCUSSION TITLE
Support for nugs.net and livephish.com

### DISCUSSION THREAD

**Discussion by tube0013** [music-assistant/support#2414] [Feature requests and ideas]
Both these services are administered by the same company (nugs) and have a vast library of live concert recordings (generally official soundboards) https://www.nugs.net/browse-artists/  livephish has a free content and paid membership with access to all their official recordings.

Happy to share captures done with Charles Proxy on iOS apps. Fairly certain the code for nugs could be adapted for livephish easily.

**Comment by JoshuaDavesMartin** [comment]
I found this request while searching for this feature myself, so I would love to see it integrated as well! It's times like these where I wish I had more time to learn how to do this myself!

**Reply by tube0013** [reply]
awesome we need to get more eyes on this. lol

**Reply by OzGav** [reply]
nugs.net has been added to MA 2.6.0beta4

For other desired providers you raise a separate feature request

**Comment by tube0013** [comment]
Also added a new request for archive.org/LMA (etree) and phish.in: https://github.com/orgs/music-assistant/discussions/3258

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
